### PR TITLE
Add docs to 5xx

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

**Related issue(s)**

#18007

### CODE REVIEW

**Review by m00g3n — DISMISSED** [review]
LGTM

File: components/central-application-gateway/README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Central Application Gateway has the following parameters:
 Central Application Gateway exposes:
 - an external API implementing a health endpoint for liveness and readiness probes
 - 2 internal APIs implementing a proxy handler accessible via a service of type `ClusterIP`
+- an endpoint for changing log level
```
Comment (grego952) [inline components/central-application-gateway/README.md:58]:
```suggestion
- an endpoint for changing the log level
```

File: components/central-application-gateway/README.md [hunk 2]
```diff
@@ -118,6 +119,14 @@ Invocation of endpoints with duplicate names results in a `400 Bad Request` fail
 - `400 Bad Request` - returned when an Application, service, or entry for the [Compass mode](https://kyma-project.io/#/01-overview/application-connectivity/README) is not specified in the path.
 - `504 Gateway Timeout` - returned when a call to the target API times out.
 
+### Debugging
+
+Log level can be changed using `zap.AtomicLevel`.
```
Comment (grego952) [inline components/central-application-gateway/README.md:124]:
```suggestion
The log level can be changed using `zap.AtomicLevel`.
```

File: docs/05-technical-reference/ac-01-application-gateway-details.md [hunk 3]
```diff
@@ -52,4 +52,6 @@ In such a case, the `Location` header is modified so that the original target pa
 The modified `Location` header has the following format: `{APP_GATEWAY_URL}:{APP_GATEWAY_PORT}/{APP_NAME}/{SERVICE_NAME}/{SUB-PATH}`.
 
 This functionality makes the HTTP clients that originally called Application Gateway follow redirects through the Gateway, and not to the service directly. 
-This allows for passing authorization, custom headers, URL parameters, and the body without an issue.
\ No newline at end of file
+This allows for passing authorization, custom headers, URL parameters, and the body without an issue.
+
+Application Gateway also rewrites all `5xx` status codes to a `502`. In such a case, the `Target-System-Status` header will contain the original code returned by the target.
```
Comment (grego952) [inline docs/05-technical-reference/ac-01-application-gateway-details.md:57]:
```suggestion
Application Gateway also rewrites all the `5xx` status codes to a `502`. In such a case, the `Target-System-Status` header contains the original code returned by the target. 
```

File: docs/05-technical-reference/ac-01-application-gateway-details.md [hunk 4]
```diff
@@ -52,4 +52,6 @@ In such a case, the `Location` header is modified so that the original target pa
 The modified `Location` header has the following format: `{APP_GATEWAY_URL}:{APP_GATEWAY_PORT}/{APP_NAME}/{SERVICE_NAME}/{SUB-PATH}`.
 
 This functionality makes the HTTP clients that originally called Application Gateway follow redirects through the Gateway, and not to the service directly. 
-This allows for passing authorization, custom headers, URL parameters, and the body without an issue.
\ No newline at end of file
+This allows for passing authorization, custom headers, URL parameters, and the body without an issue.
```
Comment (grego952) [inline docs/05-technical-reference/ac-01-application-gateway-details.md:55]: Can you also copy the changes to https://github.com/kyma-project/application-connector-manager/blob/main/docs/user/technical-reference/07-10-application-gateway-details.md ?

**Review by grego952 — APPROVED** [review]

**Pull request merged**